src/components/CompleteButton/CompleteButton.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { ICompleteButton } from './types';
+import { Icon } from '../Icon';
+
+const CompleteButton: React.FC<ICompleteButton> = ({
+    isComplete,
+    color = 'green',
+  }) => {
+    return (
+      <Icon iconName={isComplete ? 'check_circle' : 'radio_button_unchecked'} color={color} filled={isComplete} />
```

Comment:
Darle funcionalidad al componente, cuando se haga click su estado cambie. Puedes inspirarte en <input type="checkbox"> de html. Aquí un artículo de como hacerlo en React, recuerda usar el icon tal y como lo estas usando. https://retool.com/blog/learn-to-build-a-react-checkbox

Reply:
Excelente, me basé en eso. La funcionalidad ya quedó usando la base de un botón con función al dar click. Hice pruebas y funciona tanto al dar click como cambiando a mano el valor de isComplete en el storybook